examples/with-sfcc/.example.env

```diff
@@ -0,0 +1,5 @@
+SFDC_CLIENT_ID=
```

Comment:
This file should be renamed to `.env.local.example` so it won't accidentally be committed.

Reply:
We want it to be committed though since that's the example env file showing that env variables are needed

Reply:
@Nutlope, @balazsorban44 is saying to jeep the file but rename it to `.env.local.example`. So that when someone does use it and removes the `.example` part, it will say `.env.local`, which is already ignored by `.gitignore`.